tests/server/server-debug.phpt

```diff
@@ -1,9 +1,5 @@
 --TEST--
 MongoDB\Driver\Server debug output
---SKIPIF--
-<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
-<?php skip_if_not_live(); ?>
-<?php skip_if_not_clean(); ?>
```

Comment:
Only `skip_if_not_clean` should have been removed. This test still depends on a live server.

In https://github.com/mongodb/mongo-php-driver/pull/1239#discussion_r680017314 I was only referring to the particular line with `skip_if_not_clean` and not the entire SKIPIF. Apologies for not being more clear.